src/Domain/Creators/SphereTimeDependentMaps.cpp

```diff
@@ -50,6 +55,9 @@ TimeDependentMapOptions::create_functions_of_time(
   std::unordered_map<std::string, double> expiration_times{
       {size_name, std::numeric_limits<double>::infinity()},
       {shape_name, std::numeric_limits<double>::infinity()},
+      {expansion_name, std::numeric_limits<double>::infinity()},
+      {expansion_outer_name, std::numeric_limits<double>::infinity()},
+      {rotation_name, std::numeric_limits<double>::infinity()},
```

Comment:
For the sphere domain, expansion and rotation won't need expiration times because of the `SettleToConst` FoTs.

Reply:
Removed. I had to leave the one for the unused outer region.